jenkinsfiles/acm-qe.Jenkinsfile

```diff
@@ -154,10 +160,18 @@ pipeline {
                     if (params.TEST_TAGS == '@post-release') {
                         DOWNSTREAM = "--downstream false"
                     }
+
+                    // Use the '@pre-upgrade' tag to update the Submariner catalog
```

Comment:
Could you please update the message os it will be more understandable to the future reviews of the code?

// The FLOW controls execution of the "Deploy" stage.
// By default - "--deploy"
// When '@pre-upgrade' tag provided, it will use "--subm-catalog-update" to update the Submariner catalog source